kadi/commands/states.py

```diff
@@ -2532,6 +2532,9 @@ def get_continuity(
 def interpolate_states(states, times):
     """Interpolate ``states`` table at given times.
 
+    Any ``times`` that are outside the range of the states table are clipped to the
```

Comment:
Changing the times seems a bit out of scope, especially if the times are not explicitly returned.

Reply:
The times are not literally changed. I updated the docstring to be more clear.